### ISSUE TITLE
save as image after render pdf

### ISSUE THREAD

**Issue by sabdaul-ulum** [mozilla/pdf.js#6398]
can i save as image after render pdf?

**Comment by timvandermeij** [comment]
Once rendering is finished, you can save the canvas as a PNG image using `canvas.toDataURL("image/png");`. Closing as answered (and out-of-scope for this project).
